src/main/java/org/polyjdbc/core/key/AutoIncremented.java

```diff
@@ -25,12 +25,9 @@
  * @author Adam Dubiel
  */
 public class AutoIncremented implements KeyGenerator {
-
-    private static final long DEFAULT_VALUE = 0;
-
     @Override
     public long generateKey(String sequenceName, Transaction transaction) throws SQLException {
-        return DEFAULT_VALUE;
+        throw new RuntimeException("not implemented");
```

Comment:
will change to `throw new RuntimeException("Not implemented. Can't generate key on AutoIncremented");`